forms-flow-api/requirements.txt

```diff
@@ -1,71 +1,91 @@
 Brotli==1.1.0
-Flask-Caching==2.0.1
-Flask-Migrate==4.0.7
+Deprecated==1.2.18
+Flask-Caching==2.3.1
+Flask-Migrate==4.1.0
 Flask-Moment==1.0.6
 Flask-SQLAlchemy==3.1.1
-Flask==2.3.3
-Jinja2==3.1.5
-Mako==1.3.8
+Flask==3.1.0
+Jinja2==3.1.6
+Mako==1.3.9
 MarkupSafe==3.0.2
 PyJWT==2.10.1
 PySocks==1.7.1
 SQLAlchemy-Utils==0.41.2
-SQLAlchemy==2.0.36
+SQLAlchemy==2.0.39
 Werkzeug==3.1.3
-alembic==1.14.0
-aniso8601==9.0.1
-attrs==24.2.0
+alembic==1.15.1
+aniso8601==10.0.0
+attrs==25.3.0
 blinker==1.9.0
 cachelib==0.13.0
-certifi==2024.8.30
+certifi==2025.1.31
 cffi==1.17.1
-charset-normalizer==3.4.0
-click==8.1.7
-cryptography==44.0.1
-ecdsa==0.19.0
+charset-normalizer==3.4.1
+click==8.1.8
+cryptography==44.0.2
+ecdsa==0.19.1
 flask-jwt-oidc==0.7.0
-flask-marshmallow==1.2.1
+flask-marshmallow==1.3.0
 flask-restx==1.3.0
-formsflow_api_utils @ git+https://github.com/auslin-aot/forms-flow-ai.git@feature/fwf-4315-update-api-docs#subdirectory=forms-flow-api-utils
+formsflow_api_utils @ git+https://github.com/aot-technologies/forms-flow-ai.git@a2c74f2c56fa1fabfda12fa6a044b81b30905950#subdirectory=forms-flow-api-utils
+googleapis-common-protos==1.69.2
```

Comment:
whether we need `googleapis-common-protos `package?

Reply:
It's coming as a dependency as one of the packages we use, I used make setup to include the dependencies which will avoid any runtime missing package error